core/common/src/main/java/alluxio/RuntimeConstants.java

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
thanks for the explanation ... on a related note, do you guys think it would make sense to move all the constants that are property key names to a separate class (e.g. PropertyKey), I find it mildly confusing that Constants contains both property key names and various other constants. I think `conf.get(PropertyKey.VERSION)` reads better than `conf.get(Constants.VERSION)` this also avoids confusion between `Constants.VERSION` and `RuntimeConstants.VERSION`